.github/workflows/ci-tests.yaml

```diff
@@ -59,10 +59,12 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
+      - name: Setup Golang
+        uses: actions/setup-go@v3
       - name: Run golangci-lint
-        uses: golangci/golangci-lint-action@v3
+        uses: golangci/golangci-lint-action@v5
         with:
-          version: v1.52.2
+          version: v1.57.2
           args: --timeout 5m
```

Comment:
We're still seeing lint failures, though with a slightly better error message. Wondering if https://github.com/golangci/golangci-lint/issues/395#issuecomment-1690510780 could help?